src/logic/garden-logic.ts

```diff
@@ -15,12 +15,8 @@ export default function useGardenLogic() {
   } = useGardenState()
 
   const actions = useActions()
-  const [
-    proposals,
-    filters,
-    proposalsFetchedCount,
-    blockHasLoaded,
-  ] = useProposals()
+  const [proposals, filters, proposalsFetchedCount, blockHasLoaded] =
+    useProposals()
```

Comment:
since there's no change in here, can we revert the changes in this file?

Reply:
It was auto formatting thing 😅